api/client/client.go

```diff
@@ -783,6 +783,11 @@ func (c *Client) GetBotUsers(ctx context.Context) ([]types.User, error) {
 func (c *Client) GetAccessRequests(ctx context.Context, filter types.AccessRequestFilter) ([]types.AccessRequest, error) {
 	stream, err := c.grpc.GetAccessRequestsV2(ctx, &filter, c.callOpts...)
 	if err != nil {
+		err := trail.FromGRPC(err)
+		if trace.IsNotImplemented(err) {
```

Comment:
nit: overshadow in a smaller scope?

```suggestion
		if err := trail.FromGRPC(err); trace.IsNotImplemented(err) {
```

Reply:
Removed the last conversion instead to avoid doing it twice.